_includes/template/bodycontents.html

```diff
@@ -198,8 +198,9 @@ <h2 class="light">{{ page.description }}</h2>
             {{ content }}
             {% include feedback.html %}
             <p class="let-us-know">
+              {% capture issue_title %}I can't find what I'm looking for: {{ page.path }}{% endcapture %}
                 Can't find what you're looking for?
-                <a href="#" data-modal="let-us-know">
+                <a href="https://github.com/tinymce/tinymce-docs/issues/new?title={{ issue_title | url_encode }}" target="_blank" data-modal="let-us-know">
```

Comment:
Can you chuck `rel="noopener"` as an attribute here? If it's crossing out of our domain, it should probably have it.

Reply:
fixed